src/model/UserphraseModel.h

```diff
@@ -71,4 +72,5 @@ public slots:
 
     std::unique_ptr<ChewingContext, void (*)(ChewingContext*)> ctx_;
     UserphraseSet userphrase_;
+    int addResult;
```

Comment:
I initialize it in UserphraseModel.cpp:55,
or I should initialize the variable in header?